VIS/Areas/VIS/Scripts/F20/aheaderpanel.js

```diff
@@ -74,9 +74,14 @@
 
                         var colValue = getFieldValue(mField, iControl);
 
+                        if (iControl instanceof VIS.Controls.VButton) {
+                            if (!colValue && colValue.indexOf("{") > -1)
```

Comment:
Colvalue will try to check index if colValue is null. It will raise an exception.

Reply:
insert or condition